### PR TITLE
refactor: validation in community form

### PR DESCRIPTION
**Pull request by Prerna290**
## Type
- ( ) Fix
- (X) Refactor
- ( ) Style
- ( ) Docs
- ( ) Feat
- ( ) Hotfix

## Screenshots
image
image
image

### CODE REVIEW

File: apps/commudle-admin/src/app/feature-modules/community-control-panel/components/community-edit-details/community-edit-details.component.html [hunk 1]
```diff
@@ -88,6 +115,9 @@
       <div class="about" role="group">
         <label>
           About*
+          <span *ngIf="communityForm.get('community').get('about').touched">
```
Comment (arpansac) [inline apps/commudle-admin/src/app/feature-modules/community-control-panel/components/community-edit-details/community-edit-details.component.html]: Fix this please

File: apps/shared-components/tag/tag.component.html [hunk 2]
```diff
@@ -10,3 +10,6 @@
     type="text"
   />
 </nb-tag-list>
+<span *ngIf="tags.length <= 0">
```
Comment (arpansac) [inline apps/shared-components/tag/tag.component.html]: min 5 tags should be present.

**Review by arpansac — CHANGES_REQUESTED** [review]

File: apps/commudle-admin/src/app/feature-modules/community-control-panel/components/community-edit-details/community-edit-details.component.html (hunk 1 above)
Comment (Prerna290) [reply apps/commudle-admin/src/app/feature-modules/community-control-panel/components/community-edit-details/community-edit-details.component.html]: fixed

File: apps/shared-components/tag/tag.component.html (hunk 2 above)
Comment (Prerna290) [reply apps/shared-components/tag/tag.component.html]: Done

**Review by ArshdeepGrover — APPROVED** [review]

**Review by ArshdeepGrover — APPROVED** [review]

**Review by ArshdeepGrover — APPROVED** [review]

**Pull request merged**